integration_tests/features/support/steps.js

```diff
@@ -6,6 +6,7 @@ const WalletProcess = require('../../helpers/walletProcess');
 const expect = require('chai').expect;
 const {waitFor, getTransactionOutputHash} = require('../../helpers/util');
 const TransactionBuilder = require('../../helpers/transactionBuilder');
+var result;
```

Comment:
Just a better name to describe it
```suggestion
var lastResult;
```